core/src/main/scala/org/apache/spark/status/protobuf/StageDataWrapperSerializer.scala

```diff
@@ -496,7 +555,52 @@ class StageDataWrapperSerializer extends ProtobufSerDe {
       remoteBytesRead = binary.getRemoteBytesReadList.asScala.map(_.toDouble).toIndexedSeq,
       remoteBytesReadToDisk =
         binary.getRemoteBytesReadToDiskList.asScala.map(_.toDouble).toIndexedSeq,
-      totalBlocksFetched = binary.getTotalBlocksFetchedList.asScala.map(_.toDouble).toIndexedSeq
+      totalBlocksFetched = binary.getTotalBlocksFetchedList.asScala.map(_.toDouble).toIndexedSeq,
+      remoteReqsDuration = binary.getRemoteReqsDurationList.asScala.map(_.toDouble).toIndexedSeq,
+      shufflePushReadMetricsDist =
+        deserializeShufflePushReadMetricsDistributions(binary.getShufflePushReadMetricsDist)
+    )
+  }
+
+  private def deserializeShufflePushReadMetricsDistributions(
+      binary: StoreTypes.ShuffleReadMetricDistributions): ShuffleReadMetricDistributions = {
```

Comment:
Remove this method ? It appears unused (and incorrectly named if it is required)

Reply:
Removed the duplicate method, thanks.